eli5/sklearn/explain_weights.py

```diff
@@ -412,3 +414,13 @@ def _features(target_id):
             method='linear model',
             is_regression=True,
         )
+
+
+@explain_weights.register(Pipeline)
+def explain_weights_pipeline(estimator, feature_names=None, **kwargs):
+    last_estimator = estimator.steps[-1][1]
+    transform_pipeline = Pipeline(estimator.steps[:-1])
+    feature_names = transform_feature_names(transform_pipeline, feature_names)
```

Comment:
In other cases `feature_names` argument of explain_weights has higher priority than feature names extracted from estimator, i.e. it allows to override feature names completely. The way `feature_names` are used here looks fine though. But we should document this gotcha.